src/action.ts

```diff
@@ -256,6 +256,10 @@ class Action {
       core.setFailed(`Execution graph ${executionGraphId} has ${executionGraph.status.toLowerCase()}.`)
     }
     
+    if (fs.existsSync(outputsDir)) {
+      this.rmdir(outputsDir)
+    }
+
```

Comment:
We need to call this after the artifacts have been uploaded, otherwise we won't be able to push the report, and other bundle files (that's why the tests are failing right now). Note that since we're returning `baseDir` in this function, you can just remove it after the action concludes its execution.